skimage/measure/_marching_cubes.py

```diff
@@ -15,6 +17,12 @@ def marching_cubes(volume, level, spacing=(1., 1., 1.)):
     spacing : length-3 tuple of floats
         Voxel spacing in spatial dimensions corresponding to numpy array
         indexing dimensions (M, N, P) as in `volume`.
+    gradient_direction : string
+        Controls if the mesh was generated from an isosurface with gradient
+        ascent toward objects of interest (the default), or the opposite.
+        The two options are:
+        * descent : Object was greater than exterior
+        * ascent : Exterior was greater than object
 
```

Comment:
The problem is there's no general answer. Broadly speaking, if your object of interest happens to be a "well" instead of a "plateau" the user needs to select the appropriate kwarg here to get outward normals.